$MirOS: ports/devel/automake/1.9/patches/patch-lib_am_texinfos_am,v 1.1.7.1 2005/03/18 15:44:28 tg Exp $
--- lib/am/texinfos.am.orig	Tue May 24 16:39:44 2005
+++ lib/am/texinfos.am	Sun Oct  2 11:37:00 2005
@@ -122,7 +122,7 @@ endif !%?SUBDIRS%
 
 .PHONY: install-info-am
 if %?LOCAL-TEXIS%
-install-info-am: $(INFO_DEPS)
+install-info-am:
 	@$(NORMAL_INSTALL)
 	test -z "$(infodir)" || $(mkdir_p) "$(DESTDIR)$(infodir)"
 	@srcdirstrip=`echo "$(srcdir)" | sed 's|.|.|g'`; \
